libkbfs/hash.go

```diff
@@ -213,6 +214,28 @@ func (h Hash) Verify(buf []byte) error {
 	return nil
 }
 
+// MarshalJSON implements the encoding.json.Marshaler interface for
+// Hash.
+func (h Hash) MarshalJSON() ([]byte, error) {
+	return json.Marshal(h.String())
+}
+
+// UnmarshalJSON implements the encoding.json.Unmarshaler interface
+// for Hash.
+func (h Hash) UnmarshalJSON(data []byte) error {
```

Comment:
We do need it -- it's tested in `libfuse/libdokan.TestKbfsFileInfo`.  Which fails without this change, but passes with it.  You're right though, so I'm not sure how that works exactly.  I'll fix it.